packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenuItem.types.ts

```diff
@@ -98,18 +322,37 @@ export interface IContextualMenuItemStyleProps {
   theme: ITheme;
 
   /**
-   * Accept custom classNames
+   * Additional CSS class(es) to apply to the ContextualMenuItem.
    */
   className?: string;
 
   // Insert ContextualMenuItem style props below
+  disabled: boolean;
+  expanded: boolean;
+  checked: boolean;
+  isAnchorLink: boolean;
+  knownIcon: boolean;
+  itemClassName?: string;
+  dividerClassName?: string;
+  iconClassName?: string;
+  subMenuClassName?: string;
+  primaryDisabled?: boolean;
 }
 
-export interface IContextualMenuItemStyles {
-  /**
-   * Style for the root element.
-   */
+export interface IContextualMenuItemStyles extends IButtonStyles {
+  item: IStyle;
+  divider: IStyle;
   root: IStyle;
-
-  // Insert ContextualMenuItem classNames below
+  linkContent: IStyle;
+  anchorLink: IStyle;
```

Comment:
The following required fields are an addition from `IMenuItemStyles` which this aims to replace but may break backwards compat:

- `anchorLink`
- `iconColor`